### PR TITLE
Improvement/error handling

### PR DESCRIPTION
**Pull request by duhnnie**
Show messages in UI when API request fails.

### CODE REVIEW

File: src/actions/currentUser.actions.js [hunk 1]
```diff
@@ -7,6 +7,28 @@ const MESSAGE = {
   204: 'No stats'
 };
 
+const ERROR_MESSAGES = {
```
Comment (veymarjoel) [inline src/actions/currentUser.actions.js]: It seems to be better placed in the constants file..

File: src/services/resources/auth.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 
-export const CLIENT_ID = 'f1f22fa3391ef25e7d80';
-const AUTH_SERVER_URL = 'https://github-user-stalker-be.herokuapp.com/access_token';
+export const CLIENT_ID = '4f57b7543b2a015848f2';
+const AUTH_SERVER_URL = 'https://github-user-stalker-dev.herokuapp.com/access_token';
```
Comment (cinthia-neko) [inline src/services/resources/auth.js]: Why this change?

File: src/actions/currentUser.actions.js (hunk 1 above)
Comment (duhnnie) [reply src/actions/currentUser.actions.js]: This errors messages are only used by this the actions for current user, so maybe it should be kept here.
